examples/custom-image.nix

```diff
@@ -0,0 +1,43 @@
+{
+  gcpProject                     # (required) GCP project to deploy to
+, serviceAccount                 # (required) GCP service account email
+, accessKey                      # (required) path to GCP access key
+, region ? "us-east1-b"          # GCE region
+, instanceType ? "n1-standard-2" # Default GCE VM (instance) type
+, subnet ? ""
+, volumeSize ? 50                # Default volume size
+, description ? "Deploy from custom image"
+, ...
+}:
+{
+  network.description = description;
+
+  resources.gceImages.custom-image = 
+    { config, resources, lib, name, pkgs, ...}:
```

Comment:
I think none of `config, resources, lib, name, pkgs` is being used so let's remove them